Gruntfile.js

```diff
@@ -2,32 +2,152 @@
+   };
+
+   config.dist.images = join(config.dist.base, 'images');
+
    grunt.initConfig({
 
       pkg: grunt.file.readJSON('package.json'),
+      versionInfo: getCodeVersion.both(),
+      config: config,
+
+      clean: {
+         build: [ config.dist.base ],
+      },
+
+      copy: {
+         images: {
+            files: [
+               {
+                  expand: true,
+                  cwd: config.images.base,
+                  src: '**/*',
+                  dest: config.dist.images,
+               },
+            ],
+         },
+      },
 
       eslint: {
          target: config.js.all,
       },
 
+      browserify: {
+         main: {
+            src: config.js.browserMainFile,
+            dest: config.dist.js.bundle,
+         },
+      },
+
+      uglify: {
+         main: {
+            files: {
+               '<%= config.dist.js.minified %>': config.dist.js.bundle,
+            },
+            options: {
+               banner: '/*! <%= pkg.name %> <%= grunt.template.today("yyyy-mm-dd") %> <%= versionInfo %> */\n',
+               sourceMap: DEBUG,
+               sourceMapIncludeSources: DEBUG,
+               mangle: DEBUG,
+               compress: DEBUG,
+               beautify: !DEBUG,
+            },
+         },
+      },
+
+      sasslint: {
+         options: {
+            configFile: join(__dirname, 'node_modules', 'sass-lint-config-silvermine', 'sass-lint.yml'),
+         },
+         target: config.sass.all,
+      },
+
+      sass: {
+         main: {
+            files: [
+               {
+                  src: config.sass.main,
+                  dest: config.dist.css.main,
+                  ext: '.css',
+                  extDot: 'first',
+               },
+            ],
+         },
+         options: {
+            sourceMap: DEBUG,
+            indentWidth: 3,
+            outputStyle: DEBUG ? 'expanded' : 'compressed',
+            sourceComments: DEBUG,
+         },
+      },
+
+      postcss: {
+         options: {
+            map: DEBUG,
+            processors: [
+               require('autoprefixer')({ browsers: '> .05%' }), // eslint-disable-line global-require
+            ],
+         },
+         styles: {
+            src: config.dist.css.main,
+         },
+      },
+
    });
 
+   grunt.loadNpmTasks('grunt-contrib-clean');
+   grunt.loadNpmTasks('grunt-contrib-uglify');
+   grunt.loadNpmTasks('grunt-browserify');
+   grunt.loadNpmTasks('grunt-contrib-copy');
    grunt.loadNpmTasks('grunt-eslint');
+   grunt.loadNpmTasks('grunt-sass');
+   grunt.loadNpmTasks('grunt-postcss');
+   grunt.loadNpmTasks('grunt-sass-lint');
 
-   grunt.registerTask('standards', [ 'eslint' ]);
+   grunt.registerTask('standards', [ 'eslint', 'sasslint' ]);
+   grunt.registerTask('build-js', [ 'browserify', 'uglify' ]);
+   grunt.registerTask('build-css', [ 'sass', 'postcss:styles' ]);
+   grunt.registerTask('build', [ 'build-js', 'build-css', 'copy:images' ]);
```

Comment:
Can we add a `develop` task that builds and watches for changes? It should basically cover the 95% usecase for what a developer wants to do when he's working on this repo.